internal/runbits/dependencies/changesummary.go

```diff
@@ -33,16 +33,18 @@ func OutputChangeSummary(out output.Outputer, newBuildPlan *buildplan.BuildPlan,
 			v := fmt.Sprintf("%s@%s", a.Name(), a.Version())
 			addedString = append(addedLocale, v)
 			addedLocale = append(addedLocale, fmt.Sprintf("[ACTIONABLE]%s[/RESET]", v))
-		}
-		for _, i := range a.Ingredients {
-			dependencies = append(dependencies, i.RuntimeDependencies(true)...)
-			directDependencies = append(dependencies, i.RuntimeDependencies(false)...)
+
+			for _, i := range a.Ingredients {
+				dependencies = append(dependencies, i.RuntimeDependencies(true)...)
+				directDependencies = append(directDependencies, i.RuntimeDependencies(false)...)
+			}
 		}
 	}
 
 	dependencies = sliceutils.UniqueByProperty(dependencies, func(i *buildplan.Ingredient) any { return i.IngredientID })
 	directDependencies = sliceutils.UniqueByProperty(directDependencies, func(i *buildplan.Ingredient) any { return i.IngredientID })
-	numIndirect := len(dependencies) - len(directDependencies)
+	commonDependencies := directDependencies.CommonRuntimeDependencies().ToIDMap()
```

Comment:
This identifies any dependencies that are used between all ingredients. Effectively this should give us Python, without us needing to know anything about languages.

This'll likely not work very well if someone does an install of multiple requirements across different namespaces, but that's incredibly niche/unlikely.

Reply:
That's a good solution I think.